.github/workflows/infra-release.yml

```diff
@@ -0,0 +1,22 @@
+name: Release Infrastructure
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    paths:
+      - "infra/prod/italynorth/**"
+      - ".github/workflows/infra-review.yml"
+      - ".github/workflows/infra-release.yml"
+
+jobs:
+  infra_release_prod_itn:
+    uses: pagopa/dx/.github/workflows/infra_apply.yaml@2a6b15a40418406315ae881d3c5c5c1988630ae1
+    name: Release Infrastructure Prod Italy North
```

Comment:
I'd set `Prod Italy North` because GitHub concatenates the workflow name, the job name and the workflow call's job name